sdk/networkcloud/Azure.ResourceManager.NetworkCloud/tests/NetworkCloudManagementTestBase.cs

```diff
@@ -21,12 +21,19 @@ protected NetworkCloudManagementTestBase(bool isAsync, RecordedTestMode mode)
         : base(isAsync, mode)
         {
             BodyKeySanitizers.Add(new BodyKeySanitizer("properties.vmImageRepositoryCredentials.password") { Value = "fake-password123" });
+            BodyKeySanitizers.Add(new BodyKeySanitizer("..containerUrl") { Value = "https://sanitized.blob.core.windows.net/container" });
+            BodyKeySanitizers.Add(new BodyKeySanitizer("..vaultUri") { Value = "https://sanitized.vault.azure.net" });
+
+            SanitizersToRemove.Add("AZSDK3402");
         }
 
         protected NetworkCloudManagementTestBase(bool isAsync)
```

Comment:
Why are there two separate constructors? Should one of them call into the other?

Reply:
I believe this is because the [ManagementRecordedTestBase](https://github.com/Azure/azure-sdk-for-net/blob/bc7910523da63e5d6cd8be08e08d16a2358d81c3/common/ManagementTestShared/Current/ManagementRecordedTestBase.cs#L14) also has two versions

Reply:
Should we have a private Initialize method so we don't have to repeat the code between both constructors?

Reply:
That's a good idea - I can add it as a tech debt item for us in the next version if that works. We're a little behind time-wise with this API version, so the next one will be coming pretty soon